translator/source_code/deduplicate.py

```diff
@@ -0,0 +1,67 @@
+import json
+import math
+import copy
+
+
+def wzdx_input():
+    icone_input = open('../sample files/Output Message/icone_wzdx_translated.geojson', 'r').read()
+    cotrip_input = open('../sample files/Output Message/cotrip_wzdx_translated.geojson', 'r').read()
+    return json.loads(icone_input), json.loads(cotrip_input)
+
+
+def find_duplicate(icone_message_set, cotrip_message_set):
+    deduplicated_feed = copy.deepcopy(cotrip_message_set)
+    deduplicated_feed['features'] = []
+    
+    icone_message_map = {}
+    for icone_message in icone_message_set['features']:
+        icone_message_map[icone_message['properties']['road_event_id']] = icone_message  
+
+    for cotrip_message in cotrip_message_set['features'] :
+        add_cotrip = True
+
+        for icone_message in icone_message_set['features']:
+            cotrip_lat_long = cotrip_message['geometry']['coordinates'][0]
+            icone_lat_long = icone_message['geometry']['coordinates'][0]
+            distance = getDist((cotrip_lat_long[0], cotrip_lat_long[1]), (icone_lat_long[0], icone_lat_long[1]))
+
+            if distance < 100:
+                add_cotrip = False
+                del icone_message_map[icone_message['properties']['road_event_id']] 
+                deduplicated_feed['features'].append(icone_message)
+                break
+
+        if add_cotrip:
+            deduplicated_feed['features'].append(cotrip_message)
+    for id in icone_message_map:
+        deduplicated_feed['features'].append(icone_message_map[id])
+    return deduplicated_feed
+
+
+def combine_duplicate_messages(icone_obj, cotrip_obj):
+    print('duplicate message found')
+    return cotrip_obj
+
+
+
+def getDist(origin, destination):
+    if not origin or not destination:
+        return None
+
+    lat1, lon1 = origin  # lat/lon of origin
+    lat2, lon2 = destination  # lat/lon of dest
+
+    radius = 6371.0*1000  # meters
```

Comment:
We might consider updating the variable names to be more descriptive [[1](https://moderatemisbehaviour.github.io/clean-code-smells-and-heuristics/#g19-use-explanatory-variables)], e.g. `radiusInMeters`, it helps avoid mental mapping

Reply:
i deleted that file since we are not using that deduplication anymore